modules/chapter2/pages/section4.adoc

```diff
@@ -0,0 +1,141 @@
+= Assessment Lab: Working with Data Science Projects
+:navtitle: Lab
+
+In this exercise, you will practice the use of data science projects, permissions, workbenches, environment variables, and data connections.
+
+[INFO]
+====
+*Prerequisites*
+
+To perform this exercise, ensure that your platform meets the following conditions:
+
+* An S3 bucket.
+* The `data-scientists` user group exists in OpenShift.
+* The `user4` and `user5` users exist in OpenShift and are assigned to the `data-scientists` group.
+
+The identity provided of your demo platform already includes these users.
+To create the group, log in as an administrator user in OpenShift and then apply the https://github.com/RedHatQuickCourses/rhods-qc-apps/blob/ch02s04-lab/1.intro/chapter2/lab/group.yaml file.
```

Comment:
How about mentioning that you can also apply the YAML using the OCP web console YAML editor? Warn them that this YAML should be applied to the correct OCP project/namespace

Reply:
Afaik, Groups are not namespaced